limitedstore/limitedstore.go

```diff
@@ -26,6 +27,19 @@ type LimitedStore struct {
 	mutex *sync.Mutex
 }
 
+// Pair structure to perform map-sorting
+type Pair struct {
```

Comment:
I would be pleased if this struct is not exported outside of this package. However, I can do this by myself when merging this PR if you don't want to :)